extensions/wikia/Email/Controller/FounderController.class.php

```diff
@@ -209,3 +210,114 @@ protected function getFooterEncouragement() {
 			->parse();
 	}
 }
+
+class FounderActiveController extends FounderController {
+
+	/**
+	 * Define this and do nothing since we don't need the checks of our parent
+	 */
+	public function initEmail() {
+		// NOOP
+	}
+
+	public function getSubject() {
+		return $this->getMessage( 'emailext-founder-active-subject' )->text();
+	}
+
+	/**
+	 * @template multiAvatarLayout
+	 */
+	public function body() {
+		$this->response->setData( [
+			'salutation' => $this->getSalutation(),
+			'summary' => $this->getSummary(),
+			'avatarAndDetailsList' => $this->getChangeList(),
+			'buttonText' => $this->getChangesLabel(),
+			'buttonLink' => $this->getChangesLink(),
+			'contentFooterMessages' => [
+				$this->getMessage( 'emailext-founder-active-footer-1' )->text(),
+				$this->getMessage( 'emailext-founder-active-footer-2' )->text(),
+			],
+			'hasContentFooterMessages' => true
+		] );
+	}
+
+	protected function getSummary() {
+		return $this->getMessage( 'emailext-founder-active-summary' )->parse();
+	}
+
+	private function getChangeList() {
+		$changes = $this->getRecentActivity();
+		$changeList = [];
+
+		foreach ( $changes as $event ) {
+			try {
+				$eventUser = $this->getUserFromName( $event[ 'user' ] );
+			} catch ( ControllerException $e ) {
+				continue;
+			}
+
+			$changeList[] = [
+				'editorProfilePage' => $eventUser->getUserPage()->getFullURL(),
+				'editorUserName' => $eventUser->getName(),
+				'editorAvatarURL' => $this->getAvatarURL( $eventUser ),
+				'details' => $this->getDetails( $event ),
+			];
+		}
+
+		return $changeList;
+	}
+
+	protected function getDetails( array $event ) {
+		$titleKey = $event['title'];
+		$ns = $event['ns'];
+		$title = \Title::newFromText( $titleKey, $ns );
+
+		$titleUrl = $title->getFullURL();
+		$titleText = $title->getText();
+
+		$msgKey = $event['type'] == 'new' ? 'emailext-founder-new-update' : 'emailext-founder-edit-update';
+		return $this->getMessage( $msgKey, $titleUrl, $titleText )->parse();
+	}
+
+	private function getRecentActivity( $num = 5 ) {
```

Comment:
I do like the idea of having ```$event``` actually be a class that I can call ```$event->typeIsNew()``` etc etc, but its a slippery slope.  If I add ```Email\Model\RecentActivity``` with the current parameters and returned props, maybe I should consider making the returned props configurable?  If that, maybe it should be moved outside of the Email extension and handle configuring any of the parameters?  If that, maybe the ```action``` should be configurable and I make a general base Model for all types of MW API requests and different subclass models for every combination of ```action``` and ```list```.

I think that would actually be really useful (and believe me I briefly considered it) but is far beyond the scope of this ticket and we'd never get this email out ;)